components/tools/OmeroPy/src/omero/plugins/group.py

```diff
@@ -186,7 +185,9 @@ def perms(self, args):
                          % (g.name.val, gid, perms))
         else:
             try:
-                a.changePermissions(Grp(gid, False), perms)
+                chmod = omero.cmd.Chmod(
+                    type="/ExperimenterGroup", id=gid, permissions=str(perms))
+                c.sf.submit(chmod)
```

Comment:
With this semantic change, the chmod command is submitted but returns immediately ending up in a race condition for the tests in https://ci.openmicroscopy.org/job/OMERO-5.1-merge-integration-python/424/testReport/test.integration.clitest.test_group/TestGroupRoot/testPerms___perms_rwra___rwr______id_/. As suggested by @joshmoore, we might want to use `c.submit()` instead.